lib/schema-evolution-manager/apply_util.rb

```diff
@@ -35,6 +36,24 @@ def apply!(dir)
       count
     end
 
+    def persist_pgpass(password)
+      Preconditions.check_not_blank(password, "password cannot be blank")
+      puts "Creating temp pgpass at #{@pgpass_file.path}"
+      FileUtils.chmod(0600, @pgpass_file.path)
+
+      FileUtils.rm_rf("#{ENV['HOME']}/.pgpass")
```

Comment:
Actually I can also just raise a run time exception if pgpass already exists 

Yoni

Sent from my mobile

> On Mar 16, 2016, at 6:59 PM, Sean Smith <notifications@github.com> wrote:
> 
> In lib/schema-evolution-manager/apply_util.rb:
> 
> > @@ -35,6 +36,24 @@ def apply!(dir)
> >        count
> >      end
> >  
> > +    def persist_pgpass(password)
> > +      Preconditions.check_not_blank(password, "password cannot be blank")
> > +      puts "Creating temp pgpass at #{@pgpass_file.path}"
> > +      FileUtils.chmod(0600, @pgpass_file.path)
> > +
> > +      FileUtils.rm_rf("#{ENV['HOME']}/.pgpass")
> This seems unintuitive for a user that you will nuke their pgpass file. There is a PGPASSWORD environment variable, which seems safer.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>